### ISSUE TITLE
Empty edit widget doesn't show cursor any more

### ISSUE THREAD

**Issue by hpdeifel** [jtdaugherty/brick#105]
Maybe due to 26fe2d349d0878808a. It's important that even an empty widget shows its cursor to indicate that the user can type something.

Example:

```haskell
event :: Editor Text Int -> BrickEvent Int e -> EventM Int (Next (Editor Text Int))
event edit (VtyEvent (EvKey (KChar 'q') [])) = halt edit
event edit (VtyEvent ev) = handleEditorEvent ev edit >>= continue

draw :: Editor Text Int -> [Widget Int]
draw = pure . renderEditor True

app :: App (Editor Text Int) e Int
app = App { appDraw = draw
          , appChooseCursor = showFirstCursor
          , appHandleEvent = event
          , appStartEvent = return
          , appAttrMap = const $ attrMap defAttr []
          }

main :: IO ()
main = void $ defaultMain app (editor 0 (txt . T.concat) (Just 1) T.empty)
```

- expected: Cursor is always displayed
- actual result: Cursor is hidden on startup and only appears after the user types something

**Comment by jtdaugherty** [comment]
You're right! That's definitely broken. I've reverted the offending change. I'll have to think harder about how to get that right later. Thanks for reporting this, and thanks for the test program!

**Comment by jtdaugherty** [comment]
The fix for this is now released in `0.15.2`.
